company-profiles/pragma.md

```diff
@@ -0,0 +1,65 @@
+# Pragma
+
+## Company blurb
+
+[Pragma](https://pragma.co/) We began our journey 26 years ago with the mission of enhancing people's lives through the transformation of businesses and the creation of exceptional and memorable experiences across all digital channels. From Colombia, we are expanding into the rest of Latin America and the United States. We currently have operations in different countries across the continent.
+
+## Company size
+
+950
+
+## Remote status
+
+Pragma is remote-friendly. They hire applicants from Colombia, Perú, Guatemala and Panamá. [Check](https://www.pragma.com.co/trabajo-desde-casa)
```

Comment:
Id change the `They` to `We` to keep the reading consistent for the user as in the blurb, we is used